src/aliceVision/depthMap/cuda/commonStructures.hpp

```diff
@@ -14,21 +14,34 @@
 #include <sstream>
 #include <iostream>
 
-#define THROW_ON_CUDA_ERROR(rcode, message) \
-        if(rcode != cudaSuccess) {                                              \
-          std::stringstream s; s << message ": " << cudaGetErrorString(err);    \
-          throw std::runtime_error(s.str());                                    \
-        }
+#include <aliceVision/system/Logger.hpp>
```

Comment:
I don't know why it works for you.
But I have build troubles on both windows+cuda9 and linux+cuda7 when using boost includes in cuda files.
That's why I have removed all includes to logging from .cu files.